tests/h/views/api_groups_test.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import mock
+import pytest
+
+from pyramid.httpexceptions import HTTPNoContent
+
+from h.views import api_groups as views
+
+
+class FakeRequest(object):
```

Comment:
Ok; cool. I was trying to avoid depending too much on the internals of Pyramid, but I see now the view layer isn’t really a useful place to try to do that.

Also, I didn’t know about `pyramid_config.testing_securitypolicy`, so thanks for pointing that out.